src/sentry/search/eap/span_formulas.py

```diff
@@ -0,0 +1,109 @@
+from collections.abc import Callable
+from typing import Any
+
+from sentry_protos.snuba.v1.attribute_conditional_aggregation_pb2 import (
+    AttributeConditionalAggregation,
+)
+from sentry_protos.snuba.v1.endpoint_trace_item_table_pb2 import Column
+from sentry_protos.snuba.v1.trace_item_attribute_pb2 import (
+    AttributeKey,
+    AttributeValue,
+    ExtrapolationMode,
+    Function,
+    StrArray,
+)
+from sentry_protos.snuba.v1.trace_item_filter_pb2 import ComparisonFilter, TraceItemFilter
+
+
+def http_response_rate(arg: str) -> Column.BinaryFormula:
+    code = int(
+        arg  # TODO - converting this arg is a bit of a hack, we should pass in the int directly if the arg type is int
+    )
+
+    # TODO - handling valid parameters should be handled in the function_definitions (span_columns.py)
+    if code not in [1, 2, 3, 4, 5]:
+        raise Exception("http_response_rate takes in a single digit (1,2,3,4,5)")
```

Comment:
We should stick with raising `InvalidSearchQuery` when its an invalid user query